### DISCUSSION TITLE
New Team Member

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#321] [Team Posts]
Dear all,

I want to welcome Simon Schneider to our team! Simon is a Member of the Akaflieg Freiburg flight club and has already been working on a port of Enroute Flight Navigation to Apple iOS devices for a while. I should have introduced him earlier, and I apologize for not having done that.

@schneisi Welcome!

Best wishes,

Stefan.

**Comment by Lukesw74** [comment]
Hello Simon... welcome
Ciao
Luca

**Comment by ddemoll** [comment]
Hello @schneisi what issues are we facing with the port to iOS?
